plugins/trendmicro_visionone/Dockerfile

```
@@ -0,0 +1,29 @@
+FROM rapid7/insightconnect-python-3-38-plugin:4
```

We have recently updated the SDK to `v5` so kindly change this 😄 
```suggestion
FROM rapid7/insightconnect-python-3-38-plugin:5
```